_posts/2020-03-30-exploting-spatial-diversity-using-unetstack3.md

```diff
@@ -0,0 +1,98 @@
+---
+layout: post
+comments: true
+title: Exploiting Dsitributed Spatial Diversity Using UnetStack
+date: 30/03/2020
+author: Prasad Anjangi
+categories: howto
+feature-img: "assets/img/sd-unet.jpg"
+thumbnail: "assets/img/sd-unet.jpg"
+tags: [howto, spatial diversity, cooperative diversity, robustness, data rate, performance boost]
+---
+
+Though it has been a buzzword over the past several years in the terrestrial radio frequency (RF) based wireless networks, much is not talked about practical spatial diversity systems in underwater wireless networks. It has proved and delivered tangible benefits to the end-user in the terrestrial wireless networks. Can we exploit this technique to make underwater wireless networks faster and more reliable and make that long-range communication link "just work" ? With the capability to exploit distributed spatial diversity, yes you can!
```

Comment:
Is "spatial diversity systems" really a buzzword in terrestrial RF? I feel that we are shortchanging ourselves by claiming that this has been done in RF. As far as I am aware, it hasn't. And the patent process hasn't dug up anything yet either.

Reply:
I rewrote the introduction now to better present the value and the differentiation.

Reply:
> Although it has proved and delivered tangible benefits to the end-user in the terrestrial wireless networks,  much is not talked about practical spatial diversity systems in underwater wireless networks.

This simply isn't true. There's plenty of papers talking about spatial diversity systems in underwater networks. Our push here is about implementing it in a distributed fashion.

Reply:
Agreed. In the latest update, I moved this sentence as this does not fit here. I have mentioned that distributed spatial diversity does provide tangible benefits in UW link when I conclude.